Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 # Declare your gem's dependencies in shiftcommerce.gemspec.
 # Bundler will treat runtime dependencies like base dependencies, and
 # development dependencies will be added by default to the :development group.
-gem "flex_commerce_api", git: "https://github.com/flex-commerce/flex-ruby-gem.git", tag: 'v0.6.13'
+gem "flex_commerce_api", git: "https://github.com/flex-commerce/flex-ruby-gem.git", tag: 'v0.6.20.2'
```

Comment:
Updating to latest working version of https://github.com/shiftcommerce/flex-ruby-gem – we were fairly behind.